bin/test_validator

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+
+ROOT_DIR=$(dirname $0)/..
+cd $ROOT_DIR
+
+if [[ $# != 1 ]]; then
+    echo Usage: $0 PROJECT_ID
+    false
+fi
+project_id=$1
+shift
+
+site_path=udmi_site_model
+device_id=AHU-1
+
+serial_no=validator-$RANDOM
+echo Using pubber with serial $serial_no
+
+PUBBER_OUT=pubber.out
+
+pids=`ps ax | fgrep pubber | fgrep java | awk '{print $1}'`
```

Comment:
This feels heavy handed? Is this to strengthen repeated runs of this test, or to stop interference with other running tests?